src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
```

Comment:
It's recommended to use single quotes in SCSS for better consistency. Replace double quotes with single ones: url('../images/star.svg')